hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/file/ProtectedFilesUpdateSuite.java

```diff
@@ -191,9 +191,7 @@ private HapiSpecOperation[] validateAndCleanUpOps(
             getAccountBalance(account).hasTinyBars(changeFromSnapshot("preUpdate", 0))
         };
         HapiSpecOperation[] opsArray = {
-            validateOp,
-            UtilVerbs.updateLargeFile(account, fileName, fileName),
-            getFileContents(fileName).hasContents(fileName)
```

Comment:
lost a line here?

Reply:
Yes, this was intentional. Had to fix a test.